flixel/addons/display/FlxClothSprite.hx

```diff
@@ -0,0 +1,533 @@
+	 */
+	public function new(?X:Float = 0, ?Y:Float = 0, ?SimpleGraphic:FlxGraphicAsset, ?Columns:Int = 0, ?Rows:Int = 0, pinSide:Int = FlxObject.UP, crossingConstraints:Bool = false) 
+	{
+		super(X, Y, SimpleGraphic);
+		
+		this.pinSide = pinSide;
+		this.rows = Std.int(Math.max(2, Rows));
+		this.columns = Std.int(Math.max(2, Columns));
+		this.crossingConstraints = crossingConstraints;
+		
+		_drawOffset = new Point();
+		setMesh(columns, rows);
+	}
+	
+	/**
+	 * WARNING: This will remove this sprite entirely. Use kill() if you want to disable it temporarily only and reset() it later to revive it.
+	 * Used to clean up memory.
+	 */
+	override public function destroy():Void
+	{
+		points = null;
+		constraints = null;
+		_vertices = null;
+		_indices = null;
+		_uvtData = null;
+		_drawOffset = null;
+		
+		gravity = FlxDestroyUtil.put(gravity);
+		friction = FlxDestroyUtil.put(friction);
+		_meshPixels = FlxDestroyUtil.dispose(_meshPixels);
+		
+		super.destroy();
+	}
+	
+	/**
+	 * Core update loop
+	 */
+	override public function update(elapsed:Float):Void
+	{
+		updatePoints(elapsed);
+		for (i in 0...iterations) 
+		{
+			updateConstraints(elapsed);
+		}
+		
+		super.update(elapsed);
+	}
+	
+	/**
+	 * Called by game loop, updates then blits or renders current frame of animation to the screen
+	 */
+	override public function draw():Void 
+	{
+		if (_frame == null || _meshPixels == null)
+		{
+			#if !FLX_NO_DEBUG
+			loadGraphic(FlxGraphic.fromClass(GraphicDefault));
+			#else
+			return;
+			#end
+		}
+		
+		calcImage();
+		drawImage();
+		
+		if (alpha == 0 || _frame.type == FlxFrameType.EMPTY)
+		{
+			return;
+		}
+		
+		if (dirty)	//rarely 
+		{
+			calcFrame();
+		}
+		
+		for (camera in cameras)
+		{
+			if (!camera.visible || !camera.exists || !isOnScreen(camera))
+			{
+				continue;
+			}
+			
+			getScreenPosition(_point, camera).subtractPoint(offset);
+			
+			var cr:Float = colorTransform.redMultiplier;
+			var cg:Float = colorTransform.greenMultiplier;
+			var cb:Float = colorTransform.blueMultiplier;
+			
+			var simple:Bool = isSimpleRender(camera);
+			if (simple)
+			{
+				if (isPixelPerfectRender(camera))
+				{
+					_point.floor();
+				}
+				
+				_point.copyToFlash(_flashPoint);
+				camera.copyPixels(_frame, _meshPixels, _meshPixels.rect, new Point(_flashPoint.x + _drawOffset.x,_flashPoint.y + _drawOffset.y), cr, cg, cb, alpha, blend, antialiasing);
+			}
+			else
+			{
+				_frame.prepareMatrix(_matrix, FlxFrameAngle.ANGLE_0, flipX, flipY);
+				_matrix.translate( -origin.x, -origin.y);
+				_matrix.scale(scale.x, scale.y);
+				
+				if (bakedRotationAngle <= 0)
+				{
+					updateTrig();
+					
+					if (angle != 0)
+					{
+						_matrix.rotateWithTrig(_cosAngle, _sinAngle);
+					}
+				}
+				
+				_point.add(origin.x, origin.y);
+				if (isPixelPerfectRender(camera))
+				{
+					_point.floor();
+				}
+				
+				// Create a temporary frame and draw mesh's bitmapData
```

Comment:
At this line I create a temporary frame and send it to camera. I do this because render mode is not FLX_RENDER_BLIT and the bitmapData sent is not used.

Have a better alternative to solve this?